src/modules/get_suitabilities/app/get_suitabilities_usecase.ts

```diff
@@ -0,0 +1,23 @@
+import { EntityError } from '../../../shared/helpers/errors/domain_errors'
+import { Suitability } from '../../../shared/domain/entities/suitability'
+import { NoItemsFound } from '../../../shared/helpers/errors/usecase_errors'
+import { User } from '../../../shared/domain/entities/user'
+import { IScheduleRepository } from '../../../shared/domain/repositories/schedule_repository_interface'
+
+export class GetSuitabilitiesUsecase {
```

Comment:
Route name : GetSuitabilitiesByProfessor. Change the name of the classes (this was defined with FrontEnd, they will wait for GetSuitabilitiesByProfessor route)